scripts/dnsmasq-log

```diff
@@ -5,8 +5,11 @@
 
 source ${FIREWALLA_HOME}/platform/platform.sh
 
+mkdir -p $FIREWALLA_HIDDEN/config/crontab/
```

Comment:
remove this on switching off.
https://github.com/firewalla/firewalla/blame/7640baf4f51b774fee02d6b5eee21f558601e9d7/scripts/audit-stop#L18